hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/IsStillRunningService.java

```diff
@@ -86,7 +87,9 @@ public boolean isOperationExecuting(Invocation invocation) {
         } catch (Exception e) {
             invocation.logger.warning("While asking 'is-executing': " + invocation, e);
         }
-        invocation.logger.warning("'is-executing': " + executing + " -> " + invocation);
+        if (invocation.logger.isFineEnabled()) {
+            invocation.logger.finest("'is-executing': " + executing + " -> " + invocation);
```

Comment:
You check for `isFineEnabled()` but use `finest()`. There is a `isFinestEnabled()` method which should most probably be used here.